tools/android/packaging/xbmc/src/Splash.java.in

```diff
@@ -537,7 +537,11 @@ public class Splash extends Activity
     AlertDialog.Builder builder = new AlertDialog.Builder(act);
     builder.setTitle(title);
     builder.setIcon(android.R.drawable.ic_dialog_alert);
-    builder.setMessage(Html.fromHtml(message));
+    if (Build.VERSION.SDK_INT >= 24) {
+      builder.setMessage(Html.fromHtml(message, Html.FROM_HTML_MODE_LEGACY));
```

Comment:
This seems to be correct however looks like it was added solely for compatibility reasons. Isn't there a new flag (more explicit) that describes better what we want to do here without something so generic as "legacy"?
I.e. at to what point this flag won't be also deprecated in some newer API revision?

Reply:
Yes, there are more flags, however the `MODE_LEGACY` flag provides the most basic behavior and is identical to the method it replaces. It should not be deprecated in the near future.